.golangci.yaml

```diff
@@ -32,3 +32,10 @@ linters:
     - unused
     - usetesting
     
+formatters:
+  # See https://golangci-lint.run/usage/formatters for details.
+  default: none
+  enable:
+    - gofmt
+    - goimports
+    - golines
```

Comment:
I'm not sure that golines makes the code more readable based on the changes below, and gofmt and goimports should run as part of gopls. 

If you have the Go VS Code extension, it will automatically run when you save your files:
https://code.visualstudio.com/docs/languages/go

My recommendation would be to remove `golines`